stable/dex/templates/secret.yaml

```diff
@@ -40,5 +40,9 @@ stringData:
     {{- if .staticPasswords }}
     staticPasswords:
 {{ toYaml .staticPasswords | indent 4 }}
+    {{- end }}
+    {{- if .frontend }}
+    frontend:
+{{ toYaml .frontend | indent 6 }}
```

Comment:
could you please make it similar to the `oauth2` section so we'll have uniform syntax for the file?

Like that:
```yaml
frontend: {{ toYaml .frontend | nindent 6 }}
```

Reply:
Done, retested and everything looks good.